wafer/users/templates/wafer.users/profile.html

```diff
@@ -1,165 +1,192 @@
-      accept that tradeoff for simplicity here.
-      {% endcomment %}
-      <a href="{% url 'wafer_talk_edit' talk.pk %}" class="float-right btn btn-secondary btn-lg">{% trans 'Edit' %}</a>
-      <p>{{ talk.abstract.rendered|safe }}</p>
-  </div>
-  {% endfor %}
-  {% endif %}
-  {% endif %}
+    {% endif %}
+    {# Accepted talks are globally visible #}
+    {% if profile.accepted_talks.exists %}
+      <h2>{% trans 'Accepted Talks:' %}</h2>
+      {% for talk in profile.accepted_talks %}
+        <div class="card">
+          <div class="card-body">
+            <h3 class="card-title">
+              <a href="{{ talk.get_absolute_url }}">
+                {{ talk.title }}
+              </a>
+            </h3>
+            <p>{{ talk.abstract.rendered|safe }}</p>
+          </div>
+        </div>
+      {% endfor %}
+    {% endif %}
+    {% if profile.cancelled_talks.exists %}
+      <h2>{% trans 'Cancelled Talks:' %}</h2>
+      {% for talk in profile.cancelled_talks %}
+        <div class="card">
+          <div class="card-body">
+            <h3 class="card-title">
+              <a href="{{ talk.get_absolute_url }}">
+                {{ talk.title }}
+              </a>
+            </h3>
+            <p>{{ talk.abstract.rendered|safe }}</p>
+          </div>
+        </div>
+      {% endfor %}
+    {% endif %}
+    {% if profile.provisional_talks.exists %}
+      <h2>{% trans 'Provisionally Accepted Talks:' %}</h2>
+      {% for talk in profile.provisional_talks %}
+        <div class="card">
+          <div class="card-body">
+            <h3 class="card-title">
+              <a href="{{ talk.get_absolute_url }}">
+                {{ talk.title }}
+              </a>
+            </h3>
+            <p>{{ talk.abstract.rendered|safe }}</p>
+          </div>
+        </div>
+      {% endfor %}
+    {% endif %}
+    {# Submitted talk proposals are only visible to the owner #}
+    {% if can_edit %}
+      {% if profile.pending_talks.exists %}
+        <h2>{% trans 'Submitted or Under Consideration Talks:' %}</h2>
+        {% for talk in profile.pending_talks %}
+          <div class="card">
+            <div class="card-body">
+              {% comment %}
+                Because this is one of the author's pending talks, we don't
+                need to check for edit permission's on the talk explictly.
+                This doesn't show the edit button for people with 'change-talk'
+                permissions, but we accept that tradeoff for simplicity here.
+              {% endcomment %}
+              <a href="{% url 'wafer_talk_edit' talk.pk %}"
+                 class="float-right btn btn-secondary btn-lg">
+                {% trans 'Edit' %}
+              </a>
+              <h3 class="card-title">
+                <a href="{{ talk.get_absolute_url }}">
+                  {{ talk.title }}
+                </a>
+              </h3>
+              <p>{{ talk.abstract.rendered|safe }}</p>
+            </div>
```

Comment:
We should revisit this template and see if we can't reduce the duplication between the different talk lists somehow.

Out of scope for this PR, though.

Reply:
Yeah, that'd be nice.